bin/foreman-node

```diff
@@ -162,21 +162,21 @@ def enc(minion)
   http             = Net::HTTP.new(uri.host, uri.port)
   http.use_ssl     = uri.scheme == 'https'
   if http.use_ssl?
-    if SETTINGS[:ssl_ca] && !SETTINGS[:ssl_ca].empty?
+    if SETTINGS[:ssl_ca].present?
```

Comment:
I think `.present?` is a rails thing and this actually breaks the script.

Reply:
Thanks.